test/unit/transform/classes_test.py

```diff
@@ -0,0 +1,791 @@
+"""Unit tests for TableTransformer and associated software tooling.
```

Comment:
I think there should be a unit test for the `multicol_transform_fn_factory`

Reply:
I think this type of test makes perfect sense to me. All you really want to test is that you can pass a function that takes and returns a series, and it will return a function that applies that operation to every column. I don't feel like you need to test anything deeper than that.

Reply:
Okay, I think the new test satisfies this.